.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Test
+on:
+  pull_request:
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    strategy:
+      fail-fast: true
+      matrix:
+        distro: [alpine, debian, fedora, opensuse, rocky, ubuntu]
+        method: [brew, repo, standalone]
+        shell: [ash, bash, dash, ksh, zsh]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.3.0
+      - name: Test
+        env:
+          DISTRO: ${{ matrix.distro }}
+          METHOD: ${{ matrix.method }}
+          SH: ${{ matrix.shell }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd tests/linux

```

Comment:
Why use a mixture of `cd` and working-directory in windows?

Why not use working-directory everywhere?

Reply:
Good catch, let me fix that!

Reply:
Implemented, thanks @tom-reinders !